From 7327fc242b2fe08cc7050a0e85aa97b1020cf4fc Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sat, 15 Jul 2023 16:10:26 +0200
Subject: [PATCH 47/55] ipq807x: do not carry config over during sysupgrade

Removes u-boot environment config files from conffiles list to prevent configuration preservation during sysupgrade on managed devices.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/boot/uboot-envtools/Makefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/package/boot/uboot-envtools/Makefile b/package/boot/uboot-envtools/Makefile
index 61939be6de..11a32c7ba6 100644
--- a/package/boot/uboot-envtools/Makefile
+++ b/package/boot/uboot-envtools/Makefile
@@ -59,12 +59,6 @@ MAKE_FLAGS += \
 	TARGET_CFLAGS="$(TARGET_CFLAGS)" \
 	TARGET_LDFLAGS="$(TARGET_LDFLAGS)"
 
-define Package/uboot-envtools/conffiles
-/etc/config/ubootenv
-/etc/fw_env.config
-/etc/fw_sys.config
-endef
-
 define Package/uboot-envtools/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/env/fw_printenv $(1)/usr/sbin
-- 
2.34.1

